tests/cbwd/watchdog.cc

```diff
@@ -79,6 +78,268 @@ TEST(WatchdogTest, BadConfig) {
   ASSERT_EQ(std::string(str), result);
 }
 
+TEST(WatchdogTest, JsonObject) {
```

Comment:
Please add an explanation of each test. It should be of the form:
// When .....
// Then ...
// When ....
// Then...
// ...